code401challenges/src/test/java/code401challenges/stacksandqueues/StackTest.java

```diff
@@ -0,0 +1,57 @@
+package code401challenges.stacksandqueues;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+import static org.junit.Assert.assertNotNull;
+
+public class StackTest {
+    @Test
+    public void testPush() {
+        Stack stack = new Stack();
```

Comment:
Same with your Stack here, declare the type in the diamond: `Stack<Integer> stack = new Stack();`